### ISSUE TITLE
12.07. Meeting Notes

### ISSUE THREAD

**Issue by sudo-Boris** [timherzig/decomposition_learning#18]
Timm:
- Training (Fine-tuning) all branches with full loss. Use pre-trained SWIN (frozen), and pre-trained ```gt_decoder``` and ```sl_decoder``` (NOT frozen)

Boris and Mari:
- Generate occlusion targets and start pre-training for ```occ_decoder```

Janice:
- Continue writing intermediate reports
    - E.g. Describe (3D) SWIN architecture
